webpack/assets/javascripts/react_app/components/CommonParameterForm/components/values/BooleanValue.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../../common/I18n';
+
+const BooleanValue = ({ id, name, css, value, onChange, isHidden }) => {
+  const options = [
+    { label: '', value: '' },
+    { label: __('True'), value: 'true' },
+    { label: __('False'), value: 'false' },
+  ];
+  return (
+    <select
+      id={id}
+      name={name}
+      className={`without_select2 ${css} ${isHidden && 'masked-input'}`}
```

Comment:
I think `isHidden` is confusing as a name within the context of HTML because in HTML a hidden form is not shown at all. I'd suggest to rename it `isMasked` to be clear.